blocks/library/freeform/buttons/simple.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { find } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+import { Button, TinymceIcon } from 'components';
+
+class SimpleButton extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			isActive: false,
+			isDisabled: false,
+		};
+		this.mounted = true;
+	}
+
+	componentDidMount() {
+		const { button } = this.props;
+		const fnNames = [ 'onPostRender', 'onpostrender', 'OnPostRender' ];
+		const onPostRender = find( fnNames, ( fn ) => button.hasOwnProperty( fn ) );
+		if ( onPostRender ) {
+			button[ onPostRender ].call( {
+				active: ( isActive ) => this.mounted && this.setState( { isActive } ),
+				disabled: ( isDisabled ) => this.mounted && this.setState( { isDisabled } ),
```

Comment:
The disabled function is not on the 'this' object but is instead on the passed event (weird I know). If you want a button that shows this then try the list indent button (using the lists plugin).